.github/workflows/migrations.yml

```diff
@@ -0,0 +1,22 @@
+name: Checking migrations
+
+on: [push]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - uses: actions/checkout@v2
+      with:
+        token: ${{ secrets.ACCESS_TOKEN }}
+    - name: Run app inside the docker-compose with third party api calls cached
+      run: |
+        cp resources/default.ini local.ini     
+        docker-compose build
+        docker-compose -f docker-compose.yaml -f docker-compose.dev.yaml up -d
+    - name: Check migrations
+      run: docker-compose run web python manage.py migrate
```

Comment:
Le migrate va appliquer les migrations, et potentiellement afficher ` Your models in app(s): 'ifbcat_api' have changes that are not yet reflected in a migration, and so won't be applied.`. Cependant si des migrations n'ont pas été faite, cette commande ne va pas retourner un code erreur, et cette action va alors être validé :heavy_check_mark: alors qu'on voudrait qu'elle soit en erreur :x: . Je te conseil de regarder les options que tu peux fournir à migrate ainsi que makemigrations pour voir s'il n'y aurai pas un moyen d'avoir un code d'erreur s'il manque une migration. Sinon il faudra passer par un grep et un comptage :/